src/CollectionMaker.hs

```diff
@@ -64,18 +42,18 @@ mkPolicy asset _ ctx =
   where
     txInfo = scriptContextTxInfo ctx
     txInValues = [txOutValue $ txInInfoResolved txIn | txIn <- txInfoInputs $ scriptContextTxInfo ctx]
-    txOuts = txInfoOutputs txInfo
     nftValues = [assetClassValueOf val asset | val <- txInValues]
     nftSum = sum nftValues
     mintedAmount = case flattenValue (txInfoMint txInfo) of
-      [(cs, collectionTokenName, amt)] | cs == ownCurrencySymbol ctx -> amt
-      _                                                              -> 0
+        [(cs, _, amt)] | cs == ownCurrencySymbol ctx -> amt
+        _                                            -> 0
 
 policy :: AssetClass -> Scripts.MintingPolicy
 policy asset =
   mkMintingPolicyScript $
     $$(PlutusTx.compile [||Scripts.wrapMintingPolicy . mkPolicy||])
-      `PlutusTx.applyCode` PlutusTx.liftCode asset
+    `PlutusTx.applyCode`
+    PlutusTx.liftCode asset
```

Comment:
I don't understand this formatting 🤷‍♂️ . However, setting it like this will stop this formatting from getting into every PR as noise. I've asked this particular issue on IOG's tech channel, this seems to be stylish-haskell's fault, well, I guess we need to live with it.